examples/showcase/room-scale/js/grab.js

```diff
@@ -0,0 +1,89 @@
+/* global AFRAME, THREE */
+
+/**
+*
+* It handles events coming from the hand-controls.
+* It determines if the entity is grabbed or released.
+* It updates its position to move along the controller.
+*
+*/
+AFRAME.registerComponent('grab', {

```

Comment:
This component does not make something grabbable it makes an entity to grab other entities.